src/tools/cephfs/cephfs-shell

```diff
@@ -16,6 +15,10 @@ import re
 import shlex
 import stat
 
+from cmd2 import Cmd
+from cmd2 import __version__ as cmd2_version
+from packaging.version import parse as version_parse
```

Comment:
I don't think this is a core library. It will require updating the package dependencies for cephfs-shell.

Reply:
True, but setuptools uses it, so it should be present. Anyways, I've replaced it with `distutils.version.LooseVersion`